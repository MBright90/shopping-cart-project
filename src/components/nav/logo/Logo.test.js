import { render } from '@testing-library/react'
import React from 'react'

import Logo from './index'

jest.mock('@assets/images/manson-logo-red.png', () => ({
  default: 'test-image'
}))

describe('Logo', () => {
  it('renders to match snapshot', () => {
    const { container } = render(<Logo />)
    expect(container).toMatchSnapshot()
  })
})
