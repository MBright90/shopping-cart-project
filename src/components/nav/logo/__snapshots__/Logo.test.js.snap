// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders to match snapshot 1`] = `
<div>
  <img
    alt="Manson logo in red"
    src="[object Object]"
  />
</div>
`;
